profile-widget/profilescene.cpp

```diff
@@ -491,11 +491,9 @@ void ProfileScene::plotDive(const struct dive *dIn, int dcIn, DivePlannerPointsM
 	percentageAxis->updateTicks(animSpeed);
 	animatedAxes.push_back(percentageAxis);
 
-	if (calcMax) {
-		double relStart = (1.0 - 1.0/zoom) * zoomedPosition;
-		double relEnd = relStart + 1.0/zoom;
-		timeAxis->setBounds(round(relStart * maxtime), round(relEnd * maxtime));
-	}
+	double relStart = (1.0 - 1.0/zoom) * zoomedPosition;
+	double relEnd = relStart + 1.0/zoom;
+	timeAxis->setBounds(round(relStart * maxtime), round(relEnd * maxtime));
```

Comment:
I'm curious why this was done only in the calcMax case before.

Reply:
Whatever the reason - it is a bug, since it means that the full profile is shown when dragging dive handles.